cogs/msg.py

```diff
@@ -74,7 +76,22 @@ async def on_message(self, message):
         if len(result) == 0:
             return
 
-        await message.channel.send(result[0][1])
+        text = result[0][1]
+
+        def isUrl(url):
+            try:
+                result = urlparse(url)
+                return all([result.scheme, result.netloc])
+            except:
+                return False
```

Comment:
How about a `!msg memegen enable <shorthand>` and `!msg memegen disable <shorthand>` instead of determining this at runtime? We could do the URL check once then, and then just store that in a database.

Reply:
Makes sense 👍